kobo/apps/hook/views/v2/hook.py

```diff
@@ -174,13 +174,20 @@ def retry(self, request, uid=None, *args, **kwargs):
         response = {"detail": t("Task successfully scheduled")}
         status_code = status.HTTP_200_OK
         if hook.active:
-            seconds = HookLog.get_elapsed_seconds(constance.config.HOOK_MAX_RETRIES)
-            threshold = timezone.now() - timedelta(seconds=seconds)
-
-            records = hook.logs.filter(Q(date_modified__lte=threshold,
-                                         status=HOOK_LOG_PENDING) |
-                                       Q(status=HOOK_LOG_FAILED)). \
-                values_list("id", "uid").distinct()
+            threshold = timezone.now() - timedelta(seconds=120)
```

Comment:
i had another question elsewhere about the 120-second value. help me understand it :)